src/main/java/uk/gov/ons/ctp/response/action/service/impl/CollectionExerciseClientServiceImpl.java

```diff
@@ -51,6 +51,8 @@ public CollectionExerciseDTO getCollectionExercise(final UUID collectionExcercis
 
     final HttpEntity<?> httpEntity = restUtility.createHttpEntity(null);
 
+    log.info("uriComponents.toUri() = " + uriComponents.toUriString());
```

Comment:
Maybe collectionExerciseUrl=uriComponents.toUriString() for context?

Reply:
Done